### DISCUSSION TITLE
point_0702

### DISCUSSION THREAD

**Discussion by andrea29-star** [cloudsen12/dataset#257] [Roy]
"20190124T143749_20190124T144523_T19KEB"
![image](https://user-images.githubusercontent.com/54947195/147531228-14e4f55d-7bb5-46f2-bdbd-6a57f9315b2d.png)

"20190119T143751_20190119T144606_T19KEB"
![image](https://user-images.githubusercontent.com/54947195/147531363-bdef000d-b2de-4e89-956b-e601e338344a.png)

"20190916T143751_20190916T144705_T19KEB"
![image](https://user-images.githubusercontent.com/54947195/147531496-23048023-b16b-466f-a4ae-567f3e51604f.png)

"20191110T143729_20191110T144457_T19KEB"
![image](https://user-images.githubusercontent.com/54947195/147531632-efecd65a-e35c-4937-8b8c-5460ba27030d.png)

@ryali93

**Comment by AntoniCesar** [comment]
"20190119T143751_20190119T144606_T19KEB"
![Captura](https://user-images.githubusercontent.com/86502862/153066625-ab24820a-7461-4991-8a07-4ce47cdaa5d5.JPG)
